extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,14 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+
+.amp-story-shopping-tag-currency-symbol {
+  font-family: 'Poppins', sans-serif !important;
+  font-weight: 400 !important;
+  font-size: 10px !important;
+  padding-right: 2px !important;
```

Comment:
How about RTL languages?

Reply:
It was discussed for the first iteration with the PMs that we are only dealing with English for the first iteration of the shopping feature. From the discussion that originated in https://github.com/ampproject/amphtml/pull/37112#discussion_r763390753
We have opened #37126 for dealing with the currency symbol and formatting, and will deal with international languages concurrently while solving the symbol issue.

Reply:
I'm not okay with that, it is very easy to do RTL support now, and VERY hard to go back to all paddings etc later on.
Let's get it right with each PR, please.

Reply:
Ok, I'll add RTL support once #37188 is in.